RetailCoder.VBE/UI/Inspections/InspectionResultsViewModel.cs

```diff
@@ -21,20 +20,34 @@
 
 namespace Rubberduck.UI.Inspections
 {
+    public class DisplayQuickFix
```

Comment:
I think `QuickFixInfo`, or anything that doesn't sound like a method name would be a better name for a class. "Display QuickFix" reads like a verb, something isn't right.